src/TextToSpeech.cs

```diff
@@ -37,79 +35,90 @@ public static async Task<string> Run([EventHubTrigger(Constants.EventHubSender,
                     string host = Constants.AzureTextToSpeechURL;
                     string accessToken;
 
-                    if (messageBody != null)
+                    if (messageBody == null)
                     {
-                        // Add your subscription key here
-                        // If your resource isn't in WEST US, change the endpoint
-                        Authentication auth = new Authentication("https://westus.api.cognitive.microsoft.com/sts/v1.0/issueToken", CSSubscriptionKey);
-                        try
-                        {
-                            accessToken = await auth.FetchTokenAsync().ConfigureAwait(false);
-                            log.LogInformation("Successfully obtained an access token");
-                        }
-                        catch (Exception ex)
-                        {
-                            log.LogInformation("Failed to obtain an access token");
-                            return null;
-                        }
+                        log.LogInformation("No messages received from EventHub");
+                        return null;
+                    }
+
+                    // Add your subscription key here
+                    // If your resource isn't in WEST US, change the endpoint
+                    Authentication auth = new Authentication("https://westus.api.cognitive.microsoft.com/sts/v1.0/issueToken", CSSubscriptionKey);
```

Comment:
What is Authentication type from?  Can we not use Azure.Identity?

Reply:
Sample use `Authentication` type to get an `accessToken` then call REST API to interact with Azure Cognitive Speech. It means there no Azure Cognitive Speech SDK been used.
Since there no SDK used we cannot use Azure.Identity package in there to support authentication to Azure Cognitive Speech.

Reply:
Please try to use with .GetToken method of Azure.Identity.

Reply:
I tried to migrate the speech endpoint (https://eastus.api.cognitive.microsoft.com/sts/v1.0/issuetoken) to the `/.default` scope which .GetToken() method required. but it failed with exception code "AADSTS500011".
May we can update speech authentication SDK to T2 in the future once it's released.

Reply:
Okay, speech does not support AAD right now.  Keep as is.